toolchain/check/testdata/interface/no_prelude/fail_assoc_const_not_constant.carbon

```diff
@@ -8,286 +8,54 @@
 // TIP: To dump output, run:
 // TIP:   bazel run //toolchain/testing:file_test -- --dump_output --file_tests=toolchain/check/testdata/interface/no_prelude/fail_assoc_const_not_constant.carbon
 
-// --- core.carbon
-package Core;
-
-interface ImplicitAs(Dest:! type) {
-  fn Convert[self: Self]() -> Dest;
-}
-
-// --- fail_alias_to_different_interface.carbon
-library "[[@TEST_NAME]]";
-
-import Core;
-
 interface I {
-  let T:! type;
-}
-
-interface J {
-  alias U = I.T;
-  // CHECK:STDERR: fail_alias_to_different_interface.carbon:[[@LINE+7]]:13: error: cannot implicitly convert value of type `<associated entity in I>` to `type` [ImplicitAsConversionFailure]
-  // CHECK:STDERR:   fn F() -> U;
-  // CHECK:STDERR:             ^
-  // CHECK:STDERR: fail_alias_to_different_interface.carbon:[[@LINE+4]]:13: note: type `<associated entity in I>` does not implement interface `Core.ImplicitAs(type)` [MissingImplInMemberAccessNote]
-  // CHECK:STDERR:   fn F() -> U;
-  // CHECK:STDERR:             ^
+  // CHECK:STDERR: fail_assoc_const_not_constant.carbon:[[@LINE+8]]:10: error: `Core.Int` implicitly referenced here, but package `Core` not found [CoreNotFound]
```

Comment:
I *think* this is not what this test was trying to explore. Maybe switch to a different type?

Reply:
Done.